shell/tests/common/test_node_peer.rs

```diff
@@ -1,10 +1,10 @@
-/// Test node peer, which simulates p2p remote peer, communicates through real p2p socket
-use std::collections::HashSet;
 use std::net::SocketAddr;
 use std::sync::atomic::{AtomicBool, Ordering};
 use std::sync::{Arc, RwLock};
 use std::thread;
 use std::time::{Duration, SystemTime};
+/// Test node peer, which simulates p2p remote peer, communicates through real p2p socket
```

Comment:
@akoptelov 
please check all files here in common, it should start with:
```
// Copyright (c) SimpleStaking and Tezedge Contributors
// SPDX-License-Identifier: MIT
```

and in infra.rs there are unused import commented, please remove them